imports/plugins/included/product-detail-simple/client/containers/productDetailContainer.js

```diff
@@ -182,11 +210,13 @@ class ProductDetailContainer extends Component {
 
 ProductDetailContainer.propTypes = {
   media: PropTypes.arrayOf(PropTypes.object),
-  product: PropTypes.object
+  product: PropTypes.object,
+  storedCart: PropTypes.object
 };
 
 function composer(props, onData) {
   const tagSub = Meteor.subscribe("Tags");
+  const cartSub = Meteor.subscribe("Cart", Meteor.sessionId);
```

Comment:
Where is `Meteor.sessionId` coming from? Also the cart subscription is available, (for better or worse) as `Reaction.Subscriptions.Cart`, so you can just say ` if Reaction.Subscriptions.Cart.ready()`